review-notifications/src/App.css

```diff
@@ -0,0 +1,33 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.App-header {
+  background-color: #282c34;
```

Comment:
There is actually a linter for css/scss too. It's quite nice if you write a lot of styles. If you want to you can add it to your project and travis too, will make your code even cleaner. You can read about it here: https://github.com/sasstools/sass-lint